includes/admin/upgrades/upgrade-functions.php

```diff
@@ -1550,13 +1513,13 @@ function edd_upgrade_render_v30_migration() {
 						<?php wp_nonce_field( 'edd_ajax_export', 'edd_ajax_export' ); ?>
 
 						<?php if ( ! $customer_addresses_complete ) : ?>
-							<span class="edd-migration allowed" style="<?php echo ! $migration_complete ? '' : 'display: none'; ?>">
+							<span class="edd-migration allowed" style="<?php echo $migration_complete ? '' : 'display: none'; ?>">
```

Comment:
Could we update the markup and script so that the inputs are not duplicated? This is causing validation errors because of duplicate IDs. Instead of using show/hide on the allowed/unavailable elements, we could enable/disable the inputs and change classes.

Reply:
We do both but the messaging changes depending on what happens or what step gets hung up on